src/service/RetriveElementService.ts

```diff
@@ -16,10 +16,9 @@ import * as fs from "fs";
 import * as path from "path";
```

Comment:
Is there any issue with the name of this file? "RetrieveElementService.ts" would be better, right? Just because below it is spelled correctly in the export class...

Reply:
thx for spotting. I will fix it asap.

Reply:
done in 66873c3